downstream/modules/automation-hub/proc-install-ha-hub-selinux.adoc

```diff
@@ -0,0 +1,97 @@
+
+= Install a high availability (HA) deployment of {HubName} on SELinux
+
+To set up a high availability (HA) deployment of {HubName} on SELinux, create two mount points for `/var/lib/pulp` and `/var/lib/pulp/pulpcore_static`, then assign the appropriate SELinux contexts to each. The current {PlatformNameShort} installer version requires you to first add the context for `/var/lib/pulp` and running the {PlatformNameShort} installer before adding the context for `/var/lib/pulp/pulpcore_static`.
+
+.Prerequisites
+* You have already configured a NFS export on your server.
+
+.Pre-installation procedure
+. Create a mount point at `/var/lib/pulp`:
++
+----
+$ mkdir -p /var/lib/pulp/
+----
+. Create a mount point at `/var/lib/pulp/pulpcore_static`:
++
+----
+$ mkdir -p /var/lib/pulp/pulpcore_static
```

Comment:
You have to mount /var/lib/pulp, then create this directory within the mounted filesystem.